Posit Infrastructure/Overnight Sessions.md

```diff
@@ -6,13 +6,13 @@ This document aims to provide users with information on running Posit Workbench
 
 ## Background
 
-An automated process has been put in place to close sessions in Posit Workbench at 9pm every evening.
+An automated process has been put in place to close sessions in Posit Workbench at 8pm every evening.
```

Comment:
Could we leave this line as is, but add in a new sentence e.g.

`_During Daylight Saving Time, sessions will close at 8pm._`